R/doc.R

```diff
@@ -8,7 +8,6 @@ dldoc <- function # Make directlabels documentation
 ### Package directory root.
  ){
   odir <- setwd(pkgdir)
-  on.exit(setwd(odir))
```

Comment:
@Anirban166 why remove this on.exit? if there is some error we need this on.exit in order to return to the original directory the user was in before you use setwd.

Reply:
Ah then we'll need it, reverting this in a minute